apps/demo/views/partials/maincontent-left.html

```diff
@@ -0,0 +1,10 @@
+{{$content-outer}}
+{{$preloader}}{{/preloader}}
+  <div class="govuk-grid-column-two-thirds">
+    {{$validationSummary}}{{/validationSummary}}
+    <header>
+      {{#header}}<h1 class="govuk-heading-l">{{$header}}{{/header}}</h1>{{/header}}
+    </header>
+    {{$content}}{{/content}}
+  </div>
+{{/content-outer}}
```

Comment:
I think @Rhyad-Zergane-HO has fixed this now, if not can you put it into the main bit of hof

Reply:
I have removed this now as this has been fixed in hof.